azure-pipeline/build-artifacts-and-some-tests-job-steps.yaml

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
can rename file name to more specific or remove `-some-`

Reply:
can u suggest a name? I am running 1/4th of tests in the 1st job & remaining tests in the second job

Reply:
remove some